packages/driver/src/cypress/command_queue.ts

```diff
@@ -60,335 +60,337 @@ const commandRunningFailed = (Cypress, state, err) => {
   })
 }
 
-export const create = (state, timeouts, stability, cleanup, fail, isCy) => {
-  const queue = createQueue()
+export default {
```

Comment:
@JessicaSachs I'm curious about this change. Is there a reason to prefer `export default {}` over `export const`?

Reply:
Consistency with all other command registrations. I wish that we could export named functions, but Cy does so much object stubbing on modules, I made everything consistently an `export default` to avoid any foot-gunning when running tests.